### PR TITLE
document logs and metrics for quote service

### PR DESCRIPTION
**Pull request by brettmc**
Closes: https://github.com/open-telemetry/opentelemetry.io/issues/2669

### CODE REVIEW

File: content/en/docs/demo/services/quote.md [hunk 1]
```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
 
 ## Metrics
 
-TBD
+In this demo, metrics are emitted by the batch trace and logs processors. The
+metrics describe the internal state of the processor, such as number of exported
+spans/logs, the queue limit and queue usage.
```
Comment (tiffany76) [inline content/en/docs/demo/services/quote.md:97]:
```suggestion
spans/logs, the queue limit, and queue usage.
```

File: content/en/docs/demo/services/quote.md [hunk 2]
```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
...
+
+This is enabled by setting the environment variable
```
Comment (tiffany76) [inline content/en/docs/demo/services/quote.md:99]:
```suggestion
You can enable metrics by setting the environment variable
```

File: content/en/docs/demo/services/quote.md [hunk 3]
```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
...
+`OTEL_PHP_INTERNAL_METRICS_ENABLED=true`.
+
+A manual metric is also emitted, which counts the number of quotes generated,
+including attributes for the number of items and total cost.
+
+### Initializing metrics
+
+To generate metrics, we need to create an instrument such as a counter, gauge or
```
Comment (tiffany76) [inline content/en/docs/demo/services/quote.md:107]:
```suggestion
To generate metrics, you need to create an instrument such as a counter, gauge, or
```

File: content/en/docs/demo/services/quote.md [hunk 4]
```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
...
+histogram. In this example, we retrieve a meter from the globally configured
+Meter Provider, and use it to create a counter:
```
Comment (tiffany76) [inline content/en/docs/demo/services/quote.md:109]:
```suggestion
histogram. The following example retrieves a meter from the globally configured
Meter Provider and uses it to create a counter:
```

File: content/en/docs/demo/services/quote.md [hunk 5]
```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
...
+
+```php
+$counter = Globals::meterProvider()
+    ->getMeter('my-meter')
+    ->createCounter('my-counter');
+```
+
+Metrics will accumulate, and be exported periodically based on the value
+configured in `OTEL_METRIC_EXPORT_INTERVAL`.
```
Comment (tiffany76) [inline content/en/docs/demo/services/quote.md:118]:
```suggestion
Metrics accumulate and are exported periodically based on the value
configured in `OTEL_METRIC_EXPORT_INTERVAL`.
```

File: content/en/docs/demo/services/quote.md [hunk 6]
```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
...
+
+### Generate metrics
+
+An instrument provides a method to record a new value against it. In this
+example, we can `add` values to the counter:
```
Comment (tiffany76) [inline content/en/docs/demo/services/quote.md:123]:
```suggestion
An instrument provides a method to record a new value against it. This
example shows how to `add` values to the counter:
```

**Review by tiffany76 — COMMENTED** [review]
Just a few small grammar and style suggestions. Thanks!

File: content/en/docs/demo/services/quote.md [hunk 7]
```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
...
+in `OTEL_METRIC_EXPORT_INTERVAL`.
+
+### Generate metrics
+
+An instrument provides a method to record a new value against it. This example
+shows how to `add` values to the counter:
+
+```php
+$counter->add(2);
+```
```
Comment (svrnm) [inline content/en/docs/demo/services/quote.md:127]: I am not sure if this belongs here as this reads like generic documentation for metrics and not how it is done in the demo?

Comment (julianocosta89) [reply content/en/docs/demo/services/quote.md:127]: I agree with @svrnm, in this page we should explain how we are doing in the demo itself:

```
        //manual metrics 
         static $counter; 
         $counter ??= Globals::meterProvider() 
             ->getMeter('quotes') 
             ->createCounter('quotes', 'quotes', 'number of quotes calculated'); 
         $counter->add(1, ['number_of_items' => $numberOfItems]);
```

Comment (brettmc) [reply content/en/docs/demo/services/quote.md]: Have updated to refer to what the demo is doing.

**Review by julianocosta89 — APPROVED** [review]
Thank you for taking care of that @brettmc! 🤩

File: content/en/docs/demo/services/quote.md [hunk 8]
```diff
@@ -92,8 +92,34 @@ $span->addEvent('Quote processed, response sent back', [
 
 ## Metrics
 
-TBD
+In this demo, metrics are emitted by the batch trace and logs processors. The
+metrics describe the internal state of the processor, such as number of exported
+spans/logs, the queue limit, and queue usage.
```
Comment (theletterf) [inline content/en/docs/demo/services/quote.md:97]:
```suggestion
spans or logs, the queue limit, and queue usage.
```

File: content/en/docs/demo/services/quote.md [hunk 9]
```diff
@@ -92,8 +92,34 @@ $span->addEvent('Quote processed, response sent back', [
...
+
+You can enable metrics by setting the environment variable
+`OTEL_PHP_INTERNAL_METRICS_ENABLED=true`.
```
Comment (theletterf) [inline content/en/docs/demo/services/quote.md:100]:
```suggestion
`OTEL_PHP_INTERNAL_METRICS_ENABLED` to `true`.
```

**Review by theletterf — APPROVED** [review]
Approved. Just a couple nits. Thank you!

**Pull request merged**